dss-document/src/main/java/eu/europa/esig/dss/validation/DiagnosticDataBuilder.java

```diff
@@ -806,7 +806,7 @@ private XmlCertificate getXmlCertificate(Set<DigestAlgorithm> usedDigestAlgorith
 		final XmlCertificate xmlCert = new XmlCertificate();
 
 		xmlCert.setId(certToken.getDSSIdAsString());
-                xmlCert.setBase64Encoded(Utils.toBase64(certToken.getEncoded()).getBytes());
```

Comment:
I'm not sure I understood your comment @manolo1958. I added a commit with a unit test that makes sure the response is parsed. The base64 encoding is supposed to happen under the response marshal (export object to either XML or JSON) and not before. I added a soap unit test but I'm currently using this as JSON in the rest api.

If I leave it as it was before, I'd have to read the response, decode it from base64 (again, since the XML or JSON contents encode base64binary as bas64 for transporting hte byte[]). Unless it is encoded twice on purpose (which I think is unnecessary) we could remove it.